gpii/node_modules/processReporter/src/processesBridge.js

```diff
@@ -158,8 +177,12 @@ gpii.processes.findProcessesByCommand = function (that, commandName, procArray)
     if (!procArray) {
         procArray = that.getProcessList(commandName);
     }
+    var commandNameLower = that.options.ignoreCase && commandName.toLowerCase();
     return fluid.accumulate(procArray, function (aProcInfo, matchingProcs) {
-        if (aProcInfo.command === commandName) {
+        var match = aProcInfo.command === commandName ||
```

Comment:
Very succinct.  Pay no attention to my comment on the other pull request about context awareness.  This is the way to do it via a configurable option.